src/i18n/fr/nav.ts

```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
 	'core-concepts/astro-components': 'Composants',
 	'core-concepts/astro-pages': 'Pages',
 	'core-concepts/layouts': 'Layouts',
-	'guides/markdown-content': 'Markdown',
+	'guides/markdown-content': 'Contenu Markdown',
 	'guides/imports': 'Fichiers Statiques',
 	'guides/troubleshooting': 'Dépannage',
 
 	// Features
 	features: 'Fonctionnalités',
 	'guides/configuring-astro': 'Configurer Astro',
-	'guides/styling': 'CSS et Styles',
-	'guides/data-fetching': 'Récupération de Données',
+	'guides/styling': 'CSS et Style',
```

Comment:
Could we get a second francophone opinion on this @Chrissdroid @Princesseuh?

The English text is “CSS & Styling”.